docs/book/src/clusterctl/commands/upgrade.md

```diff
@@ -75,6 +75,21 @@ clusterctl upgrade apply \
 
 <aside class="note warning">
 
+<h1>Skip-level upgrades</h1>
```

Comment:
Should we call this "Skipping minor versions in upgrades"?

Reply:
Could we also have an upgrade or link in the [tasks / upgrading clusters](https://github.com/kubernetes-sigs/cluster-api/blob/329753cae2c23d0f29a764870dc99972a50036b0/docs/book/src/tasks/upgrading-clusters.md?plain=1#L13-L14) to this?

Reply:
@oscr I think this is a different topic. This PR is about skipping Cluster API minor versions while what you referenced is about skipping Kubernetes minor versions.